timesheet_task/__openerp__.py

```diff
@@ -23,7 +23,7 @@
  'author': "Camptocamp,Odoo Community Association (OCA)",
  'maintainer': 'Camptocamp, Acsone SA/NV',
  'category': 'Human Resources',
- 'depends': ['project', 'hr_timesheet_invoice'],
+ 'depends': ['project', 'analytic_user_function'],
```

Comment:
That's good but the signature is already changed by first-party module so I don't see how `web_context_tunnel` can help. Could you be more specific?